android/app/src/main/AndroidManifest.xml

```diff
@@ -94,6 +94,9 @@
 
       <!-- RNPushNotification -->
       <meta-data android:name="com.dieam.reactnativepushnotification.notification_color" android:resource="@android:color/white"/>
+      <meta-data android:name="com.dieam.reactnativepushnotification.notification_foreground"
```

Comment:
1. I see BGFetch stull registered here
2. I see some configs are still missing here and in the gradle file. Please re-check setup carefully https://wix.github.io/react-native-notifications/docs/installation-android https://www.npmjs.com/package/react-native-push-notification

Reply:
done